include/ceed/ceed.h

```diff
@@ -382,11 +383,14 @@ CEED_EXTERN int CeedBasisGetInterp1D(CeedBasis basis, const CeedScalar **interp_
 CEED_EXTERN int CeedBasisGetGrad(CeedBasis basis, const CeedScalar **grad);
 CEED_EXTERN int CeedBasisGetGrad1D(CeedBasis basis, const CeedScalar **grad_1d);
 CEED_EXTERN int CeedBasisGetDiv(CeedBasis basis, const CeedScalar **div);
+CEED_EXTERN int CeedBasisGetCurl(CeedBasis basis, const CeedScalar **curl);
 CEED_EXTERN int CeedBasisDestroy(CeedBasis *basis);
 
 CEED_EXTERN int CeedGaussQuadrature(CeedInt Q, CeedScalar *q_ref_1d, CeedScalar *q_weight_1d);
 CEED_EXTERN int CeedLobattoQuadrature(CeedInt Q, CeedScalar *q_ref_1d, CeedScalar *q_weight_1d);
 CEED_EXTERN int CeedQRFactorization(Ceed ceed, CeedScalar *mat, CeedScalar *tau, CeedInt m, CeedInt n);
+CEED_EXTERN int CeedHouseholderApplyQ(CeedScalar *mat, const CeedScalar *Q, const CeedScalar *tau, CeedTransposeMode t_mode, CeedInt m, CeedInt n,
```

Comment:
These purely linear algebra functions should probably not be here. I can fix that in a separate PR though.

Reply:
That's what I figured, but I thought better at least to be all together. I can make a separate MR to move them to backend.h if you want?

Reply:
Go for it. Some tests will need the `ceed/backend.h` header to compile, but that's not a big fix

Reply:
https://github.com/CEED/libCEED/pull/1165